FRW/Modules/Login/ViewModel/KeyStoreLoginViewModel.swift

```diff
@@ -33,6 +33,12 @@ class KeyStoreLoginViewModel: ObservableObject {
     @Published
     var wallet: FlowWalletKit.Wallet? = nil
 
+    @Published
+    var errorJSON: String = ""
+
+    @Published
+    var errorPassword: String = ""
```

Comment:
nit: these could be combined into a struct, exposing methods to operate on the two properties instead of doing that in the view model.